test/e2e/common/node/security_context.go

```diff
@@ -633,6 +634,39 @@ var _ = SIGDescribe("Security Context", func() {
 	})
 })
 
+var _ = SIGDescribe("User Namespaces for Pod Security Standards [Feature:UserNamespacesSupport] [Feature:UserNamespacesPodSecurityStandards] [LinuxOnly]", func() {
```

Comment:
should you use the newly added valid feature variable here (`UserNamespacesPodSecurityStandards`)?

Reply:
Only `[Feature:UserNamespacesPodSecurityStandards]`? I was assuming that we use `[Feature:UserNamespacesSupport]` as well because other tests are referring to it, too.